### PR TITLE
Sw 74 ratepay direct debit

### PR DESCRIPTION
**Pull request by fatchip-Stefan**

### CODE REVIEW

File: Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Payment/Preauthorize.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ public function preauthorize(Payone_Api_Request_Preauthorization $request)
             $this->validateRequest($request);
 
             $requestParams = $this->getMapperRequest()->map($request);
-
+            unset($requestParams['company']);
```
Comment (fjbender) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Payment/Preauthorize.php]: Why is this unset here?

**Comment by fatchip-Stefan** [comment]
this was not intended to be committed an should be removed.

File: Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Payment/Preauthorize.php (hunk 1 above)
Comment (fatchip-Stefan) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Payment/Preauthorize.php]: this was not intended to be comitted and should be removed

Comment (fatchip-Stefan) [inline Frontend/MoptPaymentPayone/Components/Payone/Api/Service/Payment/Preauthorize.php]: I will update my PR

**Pull request merged**